README.md

```diff
@@ -1,5 +1,7 @@
 # CardScan
 
+[![](https://jitpack.io/v/Anwera64/cardscan-android.svg)](https://jitpack.io/#Anwera64/cardscan-android)
```

Comment:
this is not a repository I'm aware of. is this well maintained? who is keeping this up-to-date?